daemon/cmd/daemon.go

```diff
@@ -667,6 +667,10 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup,
 
 	d.redirectPolicyManager = redirectpolicy.NewRedirectPolicyManager(d.svc)
 	if option.Config.BGPAnnounceLBIP || option.Config.BGPAnnouncePodCIDR {
+		log.WithField("url", "https://github.com/cilium/cilium/issues/22246").
+			Warn("You are using the legacy BGP feature, which will only receive security updates and bugfixes. " +
+				"Its recommended you migrate to the BGP Control Plane feature if possible, which enjoys better support.")
```

Comment:
```suggestion
				"It is recommended to migrate to the BGP Control Plane feature if possible, which has better support.")
```
`Its` is incorrect here, the other changes are just suggestions that you can ignore if you prefer.